.github/workflows/image-scan.yml

```diff
@@ -95,7 +95,7 @@ jobs:
     steps:
       - name: Call Aqua scan
         id: run-scan
-        uses: portworx/workflow-image-scan/.github/actions/action-aqua-scan@v2.0.0
+        uses: portworx/workflow-image-scan/.github/actions/action-aqua-scan@v2.0.1
```

Comment:
I think this might be better with a relative path when it's in the same repo instead of pointing at a tag.
With the tag, you get this weirdness where you make it point at a version which doesn't actually exist yet.

Reply:
Lol I actually raised this in https://github.com/portworx/workflow-image-scan/pull/2#discussion_r1003188841 already. Maybe there is some merit behind adding the checkout if I keep bumping into this, not sure.

Reply:
Honestly, that's a very good point and much more convenient.

Reply:
I can only agree. Rado, if you would like to change it then please do. :+1:  :smile: